todo-app/src/store/reducer/todo.js

```diff
@@ -0,0 +1,35 @@
+import uuid from 'uuid';
+import * as actions from '../action/actions';
+
+const initialState = {
+  todo: [],
+};
+let todo;
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case actions.ADDTODO:
+      todo = { id: uuid(), completed: false, value: action.todo };
```

Comment:
This is fine but probably not the best way to do it. Your reducers functions should be pure and this reassignment here is causing a side effect. Even if it's working, not having pure functions in your reducers is always a bad idea and could potentially lead to hard to track bugs.  
In my own opinion it's as well a good idea when possible to take out the logic from your reducers. So it could be a good idea that the action.payload already has the new todo Object (instead of just a part of it). This could reduce the amount of code needed in your reducers.